pymodbus/server/async_io.py

```diff
@@ -100,11 +100,13 @@ def __init__(self, owner):
 
     def _log_exception(self):
         """Show log exception."""
+        if not self.running:
+            return
```

Comment:
Log exception can occur before and especially after server is running.

Reply:
The _log_exception() is only called twice, in handle() when self.running is True and in connection_lost(), so always after server is running. But maybe i missunderstand something

Reply:
_log_exception can be called from anywhere, we add/remove _log_exception often. Apart from that it is not logical nor correct that _log_exception depends on the state of the server.